sage-ios/SAGE/SAGE/StartSemesterView.swift

```diff
@@ -0,0 +1,129 @@
+//
+//  StartSemesterView.swift
+//  SAGE
+//
+//  Created by Andrew Millman on 1/17/16.
+//  Copyright © 2016 Cal Blueprint. All rights reserved.
+//
+
+class StartSemesterView: UIView {
```

Comment:
inherit from the formview class I made in my last PR! it removes some repeated things